### ISSUE TITLE
sickrage gui wont load

### ISSUE THREAD

**Issue by TH3-M0L3** [lsellens/service.downloadmanager.SickPotatoHead#2]
running on openelec 5.0.8 generic/x86_64

Sickpotatohead version 1.5.1

All three programs were working fine. After the update, I can no longer access sickrage via the web gui. I restored to last backup and still no joy

Couchpotato also keeps adding the same two movies multiple times to the wanted list.
"Straight outa Compton" was added around 30 times. Despite me marking it as complete on all 30, it keeps adding it every time i reboot the system.
"No escape" isnt so bad, but keeps coming back 2 or 3 times 

error log - http://pastebin.com/fuZXuJ7X

**Comment by Zkiwi** [comment]
Ditto to the above. New CP release looks much nicer but Sickrage now unable to be accessed via WebUI

**Comment by TH3-M0L3** [comment]
The issue is covered on the openelec forum. 

You have to manually modify a file. 
I found that I also had to turn auto
Updates otherwise the issue came back. 

I'm back up and running again

**Comment by TH3-M0L3** [comment]
http://openelec.tv/forum/80-sabnzbd-suite/78152-unable-to-bring-up-sickrage-page-after-autoupdate

**Comment by Zkiwi** [comment]
Thanks Dude,
I'm on there now having a read. Hopefully it will get me back up and running.

**Comment by TH3-M0L3** [comment]
No worries fella. 
Had me busy for hours before I found the thread trying to figure out why it wasn't working. 

I did the mod, and SR started, but ten auto updated and broke again. 
Disabled the auto update, and it fine. Even after restarting OE, I had to Manually start SR from command line. When that instance of putty was closed, SR stopped. 

Restarted from putty, then restarted SR from GUI and it is fine now. 

Only issue I now have is that my CP wanted list is old.

**Comment by Zkiwi** [comment]
I feel ya pain, i was hunting for the last couple of days. Unfortunately that hasn't fixed things for me. I can get CP fine but unable to load SickRage Webui still. Time to do some more digging.

Cheers for the steer towards a possible answer.

**Comment by TH3-M0L3** [comment]
Did you read the whole thread?

Someone else came in on page three with an alternative edit to the same bit of text.

It depends on the hardware you're running as to what the solution is. 
Also, beware not to use the "tab" key for spacing as this throws up an error.

**Comment by atnpgo** [comment]
Not the same person but I tried everything from that thread on my installation. Nothing has worked.

**Comment by TH3-M0L3** [comment]
it didn't work for me first time either to be honest. 

First try editing the sickbeard.ini file so it disables auto update. 
Also, once I did that, I got an error when I tried starting sickbeard.py that it couldn't find config.ini so it was using defaults. For me, that meant the IP address it got assigned clashed with SAB. 
I then had to copy over and rename my sickrage.ini to config.ini into the directory where it was looking for congfig.ini (it says where that is in the error report in putty

**Comment by atnpgo** [comment]
Thanks, I'll try that when I get home.

**Comment by Zkiwi** [comment]
Yep tried both of those. I thought the second option should work better than the first as i am running OpenElec on a Cubox so not X86 version. I might try ARM7 in place of X86. Once i have some time i will do some digging.

**Comment by atnpgo** [comment]
So due to unrelated circumstances, my HTPC was bricked. I've since rebuilt it and reinstalled from scratch.

After doing that, replacing

``` python
    from distutils.util import get_platform
    plat = get_platform()
```

by

``` javascript
    try:
        from distutils.util import get_platform
        plat = get_platform()
    except ImportError:
        import platform
        plat = platform.platform()
```

in the file `/storage/.kodi/addons/service.downloadmanager/SickPotatoHead/resources/sickbeard/pkg_resources.py` seems to have fixed my problem.

Now off to rebuild my library...

**Comment by Zkiwi** [comment]
Well two days on and loads of swearing and cursing and I am still no closer to a solution. I have tried all, the above solution, specifying Linux-x86_64 and armv7l ( which should have worked best as I am running Openelec and Sickpotatohead on a IMX6 Cubox).

I have tried loading the relevant files out of an older version of sickpotatohead but no joy.

Completely stumped now. I might try loading an older version completely and see if that will work.

I know I am missing something really simple here.
